wp-content/plugins/core/src/Blocks/Fields/CTA.php

```diff
@@ -0,0 +1,68 @@
+<?php declare(strict_types=1);
+
+namespace Tribe\Project\Blocks\Fields;
+
+use Tribe\Libs\ACF\Field;
+use Tribe\Libs\ACF\Field_Group;
+
+class CTA {
```

Comment:
This would be better as a Trait with an Interface instead of introducing global scope via static dependencies across the framework. I'll reach out on slack to clarify.

Reply:
We discussed this but we held off since you and Tanner are working on way to handle this like the color background work you're doing. Is this done in a way where we can emulate yet?

Reply:
After reviewing your other code, It's more for global settings, not a reusable field group. Did you and Lindsey have a moment to connect?

Reply:
Yes, we connected yesterday - I just pushed up a change with refactoring it using an interface/trait to make it more testable/dependable.